### PR TITLE
Update getting started guides

### PR DESCRIPTION
**Pull request by HotelCalifornia**
remove information pertaining to the beta period and update installation guides.

I'd appreciate it if someone could verify my RST-- I'm not sure I got everything right, and I still haven't been able to get a stable environment working to test locally.

### CODE REVIEW

**Comment by baylessj** [comment]
I can build it and verify it if you'd really rather not set up a build environment, but it is as simple as running `pip install -r requirements.txt` and then `make`

File: v5/getting-started/linux.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
+===================
+Installing on Linux
+===================
+
+Installing the toolchain
+------------------------
+
+1. Follow the instructions found `here <https://launchpad.net/~team-gcc-arm-embedded/+archive/ubuntu/ppa>`_ to add and install the latest version of the GNU Arm Embedded toolchain.
```
[the remark this answers is not in the document]
Comment (Octogonapus) [reply v5/getting-started/linux.rst]: Actually, I am not so sure anymore. Let me test this on a VM.

[the remark this answers is not in the document]
Comment (Octogonapus) [reply v5/getting-started/linux.rst]: It installed 7.3.1 so it will work.

File: v5/getting-started/linux.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+1. Follow the instructions found `here <https://launchpad.net/~team-gcc-arm-embedded/+archive/ubuntu/ppa>`_ to add and install the latest version of the GNU Arm Embedded toolchain.
+
+.. note:: If you are using a non-Debian-based distribution of Linux, check your favorite package repository for an updated version of the toolchain. The main requirement is that you get one that uses GCC version 7.2 or greater.
+
+Installing the CLI
+------------------
+
+1. If you do not already have one installed, install a version of Python greater than or equal to 3.6
+2. Check the latest version of the PROS CLI on `our releases page <https://github.com/purduesigbots/pros-cli3/releases/latest>`_, and run :code:`pip3 install https://github.com/purduesigbots/pros-cli3/releases/download/3.X.X/pros_cli_v5-3.X.X-py3-none-any.whl`, replacing the Xs with the numbers you found before.
```
Comment (Octogonapus) [inline v5/getting-started/linux.rst]: Also for what it's worth, this never works for me. I always have to do `sudo python3.6 -m pip install ...`.

**Comment by HotelCalifornia** [comment]
> I can build it and verify it if you'd really rather not set up a build environment, but it is as simple as running pip install -r requirements.txt and then make

yeah the last time I tried it there were a whole bunch of problems

**Pull request merged**